result-table/compare_layout.html

```diff
@@ -95,8 +93,8 @@ <h3 id="right-label">After rewrites</h3>
     function setLabels() {
         var leftLabel = getParameterByName("left-label") || "A";
         var rightLabel = getParameterByName("right-label") || "B";
-        $("#left-label").text(leftLabel);
-        $("#right-label").text(rightLabel);
+        document.getElementById("left-label").innerText=leftLabel;
```

Comment:
nitpick: space between the assignment operator `innerText = leftLabel` vs `innerText=leftLabel`; same with the next line